tests/drillhole_v4_0_test.py

```diff
@@ -16,6 +16,7 @@
 #  along with geoh5py.  If not, see <https://www.gnu.org/licenses/>.
 
 # pylint: disable=R0914
+# pylint: disable=R0401

```

Comment:
Can you put both disables on the same line, and use the word style instead of the code?

Reply:
yep, removed R0401 since the circular import problem is fixed..